Moves the botanogenetic shears to the botany node and locks it behind a botany experiment

SovietJenga opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

With the new addition of advanced engineering being locked behind bz gas shell experiment it has become even harder for botanists to get their hands on this tool which does something only itself can do.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

It was already weird to me that botanogenetic shears was in experimental tools because that node is for tier 2 tools which do the same thing other tools do but more compact/fast/less risky and simpler. So as to not make it now a tool that won't get researched basically ever due to most people's unwillingness to do toxins I'm moving this tool to it's own node.

I'm currently only moving it to it's own node because it was on a seperate one in the beginning but if I can get the go ahead i wouldn't mind moving it into the botany node. (I got the go ahead)

<!-- Argue for the merits of your changes and how they benefit the game, especially if they are controversial and/or far reaching. If you can't actually explain WHY what you are doing will improve the game, then it probably isn't good for the game in the first place. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
balance: Moves the botanogenetic shears into the botany research node and locks it behind a botany experiment
balance: Removes the advanced engineering node requirement from the botany node
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

SovietJenga committed:
Update all_nodes.dm

Fikou commented:
we have botanical experiments, this should use that

SovietJenga commented:
No problem I will add a botany experiment lock

SovietJenga committed:
Update all_nodes.dm

SovietJenga committed:
Merge branch 'tgstation:master' into master

vincentiusvin commented:
I am in favor of this 👍 (if that even means anything). Just considered it out of scope in #66738.

SovietJenga commented:
Like I said in the pull if i get the go ahead (from who the fuck even overseers botany code???) I wouldn't mind adding shears to the botany node and just locking that. Only reason it is in a different node is because it was originally in a different one.

vincentiusvin commented:
Caught skimming 🥶

SovietJenga commented:
> Caught skimming 🥶
I'm watching your every move

Dmeto commented:
> Like I said in the pull if I get the go ahead (from whoever the fuck even oversees botany code???) I wouldn't mind adding shears to the botany node and just locking that. Only reason it is in a different node is because it was originally in a different one.

MrMelbert i belive is your guy for Hydroponics
@MrMelbert

SovietJenga commented:
> What Melbert said

This pr was mostly me looking at code and seeing a common pattern and understanding it. I am not a coder and I don't have any experience coding but if it's just making text more clear I don't think it will be a problem but making a new experiment is probably beyond me but I will see what I can do because botany tools need some love.

SovietJenga commented:
Should I do wild plant mutation experiment or trait scan experiment? because I think maybe traits is actually better as a lock for nodes because of it requiring multiple scans and doesn't make me go into code and make a mess of a description trying to explain wild mutation as mutation that happen on harvest (unless im also wrong) if I can read the code right.

SovietJenga commented:
I have an idea for a description such as "Nanotrasen have recently observed a phenomenom of plants mutating upon harvest. They require you to send a scan of a sample for further investigation." (Is this grammaticly correct?) (also I ask a lot of questions damn)

SovietJenga committed:
these plants are wild

changed wild plants experiment description

SovietJenga committed:
I got the go ahead

merged the nodes

SovietJenga committed:
I can't believe I almost put g after h

SovietJenga commented on code/modules/research/techweb/all_nodes.dm:
```diff
@@ -1294,9 +1294,11 @@
 		"hydro_tray",
 		"portaseeder",
 		"seed_extractor",
+		"gene_shears",
```
done

SovietJenga committed:
fixing my bad gammar

has not have

SovietJenga committed:
again my english *cry

SovietJenga committed:
This is wild

Brought back old flavor text for the experiment and made clear. Suprising how easy it was.

Jolly-66 commented:
Glancing at this I would've questioned it

But it seems good enough

ArcaneMusic commented:
Originally I but the supershears in the experimental tools node purely because it was before we had ordinances and before experiments too, simply because people were more likely to go out of their way for experimental tools than getting the dedicated botany node. The recent addition of gas shell experiments makes this a clean change.

Tweaking the botany experiments to either be clearer or easier to follow would be nice too, considering I mainly added the botany experiment as a boost to the node and to tool around with the experiments backend.
This seems fine.

LemonInTheDark commented:
Assertion: experiments should be mandatory, but they should also be "good"
If we keep em optional, they're just gonna suck forever

bluetshirtguy commented:
Botany experiments are very unclear and obscure, I never knew they existed until someone from science bothered me for it, sci and engineering suck. Is there a way to make it more clear to botany players that this experiment exists in the first place?

vincentiusvin commented:
A prereq for the botany node as it stands now is adv engi, I believe you might not want this?

TheSilverNuke commented:
> 

Experiments should always be optional though, at least if it's something that affects a department other than your own. You shouldn't be able to cockblock the entire crew just because you can't be bothered to do toxins.

LemonInTheDark commented:
> > Assertion: experiments should be mandatory, but they should also be "good"
> > If we keep em optional, they're just gonna suck forever
> 
> Experiments should always be optional though, at least if it's something that affects a department other than your own. You shouldn't be able to cockblock the entire crew just because you can't be bothered to do toxins.

Disagree

TheSilverNuke commented:
Okay, let me rephrase it. I'd be inclined to agree with your take if TG players weren't prone to either tide, die, go SSD, or murderbone at the drop of a hat.

Players want to play the game, not do a repetitive task over and over that is actually super boring. So they'll end up doing their own gimmicks over doing what you want them to do.

SovietJenga commented:
> Botany experiments are very unclear and obscure, I never knew they existed until someone from science bothered me for it, sci and engineering suck. Is there a way to make it more clear to botany players that this experiment exists in the first place?

One of the reasons you probably have never heard of them is because botany experiments are only used in 2 nodes, one of which is questionable (experiemental surgery), and both are a discount experiments so most people don't bother actually doing them and just take the extra cost. The other reason is because the descriptions for both of them are really vague so even those that do want to do them mostly don't know what they are asking for and I need to think how I'm going to make which traits for the experiment are needed clear without just giving a list of traits.

SovietJenga committed:
more experiment clearning

SovietJenga committed:
I'm fucking stupid

This experiment works differently from what I remembered it was.

SovietJenga committed:
Merge branch 'tgstation:master' into master

the-orange-cow commented:
Love the idea behind this pr.  However, I noticed that the botany node is still locked behind advanced engi.  Now that the wild mutation experiment is a mandatory instead of merely granting a discount, would it be reasonable to remove the adv_engi requirement?

SovietJenga committed:
Bye adv engi

I miss a lot

BebeYoshi commented:
I might be crazy or something, but wouldn't be better if a the wild mutation was the required experiment and the trait one the discount one? You can get wild mutation in any round before the experiment node is researched, while for the trati you need to wait for science to research biotech (which I think unlocks the trait experiment?) so you can know the plant you need.... Also one of the traits is locked behind hacked vendor (Shadow trait) and one of them needs a lavaland plant (Sticky trait), even though most botanist hack their vendors roundstart anyways it's kind of annoying to lock more stuff behind it.

Or maybe just change the list of traits acceptable by the experiment to make them more simple, like removing shadow and sticky and replacing by slippery and laughing trait or something like that

SovietJenga commented:
>Also one of the traits is locked behind hacked vendor (Shadow trait) and one of them needs a lavaland plant (Sticky trait), even though most botanist hack their vendors roundstart anyways it's kind of annoying to lock more stuff behind it.

Valid point, I'll swap them.

SovietJenga committed:
Swapped the experiments

valid point from bebe with some traits being hard to aquire.

SovietJenga committed:
Merge branch 'tgstation:master' into master

SovietJenga committed:
r then d then r

changed the sequence of lines to fit like other nodes
